app/views/my_modules/_repositories_dropdown.html.erb

```diff
@@ -13,6 +13,27 @@
       <% else %>
         <%= truncate(repository.name) %>
         <span class="assigned-items-counter"><i class="fas fa-spinner fa-spin"></i></span>
+      <% end %>
+          <%= inventory_shared_status_icon(repository, current_team) %>
+    </a>
+  </li>
+<% end %>
+
+
+<% Repository.used_on_task_but_unshared(@my_module, current_team).each do |repository| %>
+  <li>
+    <a class="disabled dropdown-item <%= "active-item" if @repository == repository %> help_tooltips"
```

Comment:
i think we can remove this `active-item` condition, because this repository will never been active.